spring-cloud-sleuth-instrumentation/src/main/java/org/springframework/cloud/sleuth/instrument/web/SleuthWebSpan.java

```diff
@@ -78,4 +78,74 @@ public String getKey() {
 
 	}
 
+	/**
+	 * Tags related to security.
+	 *
+	 * @author Marcin Grzejszczak
+	 * @since 3.1.0
+	 */
+	enum SecurityTags implements TagKey {
```

Comment:
Is there a way that we can add a flag that shows if the principal (user) is `anonymous` (authenticated)?
Or will the list of authorities contain that information?